tools/sharkbait/src/sharkbait/permissions.clj

```diff
@@ -21,9 +22,10 @@
   "Sets the allowed actions for a permission definition."
   [permission-def]
   (let [delegate (.getAttributeDefActionDelegate permission-def)]
-    (.configureActionList delegate ["read" "write" "own"])
-    (add-implied-action delegate "own" "write")
-    (add-implied-action delegate "write" "read"))
+    (.configureActionList delegate [read write own originator])
+    (add-implied-action delegate originator own)
```

Comment:
I guess I don't see the reason for this permission, but then again I don't see how it would hurt having it.
Is it supposed to prevent the originator from losing `own` permissions by other owners?